src/bundle.rs

```diff
@@ -292,47 +530,50 @@ class recipe(A.recipe):
         res = []
         if gid in self.gid_to_connections:
             for src, dec, syn, loc, w, d in self.gid_to_connections[gid]:
-                conn = A.connection((src, A.cell_local_label(f'sd@{{dec}}', A.selection_policy.round_robin)), A.cell_local_label(f'syn_{{syn}}@{{loc}}', A.selection_policy.round_robin), w, d)
+                conn = A.connection((src, A.cell_local_label(f'sd@{dec}', A.selection_policy.round_robin)), A.cell_local_label(f'syn_{syn}@{loc}', A.selection_policy.round_robin), w, d)
                 res.append(conn)
         return res
 
     def event_generators(self, gid):
         res = []
         if gid in self.gid_to_inputs:
             for seg, frac, inp in self.gid_to_inputs[gid]:
-                tag = f'(on-components {{frac}} (region \"{{seg}}\"))'
+                tag = f'(on-components {frac} (region \"{seg}\"))'
                 if inp in self.poisson_generators:
                     syn, avg, wgt = self.poisson_generators[inp]
-                    res.append(A.event_generator(f'syn_{{syn}}@seg_{{seg}}_frac_{{frac}}', wgt, A.poisson_schedule(0, avg, gid)))
+                    res.append(A.event_generator(f'syn_{syn}@seg_{seg}_frac_{frac}', wgt, A.poisson_schedule(0, avg, gid)))
                 if inp in self.regular_generators:
                     raise \"oops\"
         return res
 
+if len(sys.argv) != 2:
+    print('Usage: main.py <network-name>')
```

Comment:
Yeah ... I see why might seem appealing, but having things be done silently is almost never good. 
1. Assume the user _did_ expect a simulation/network `A` but something went wrong, it is not written, and we just run `B`, since now only one is found
2. Assume a name change in the XML, suddenly a different network is run. Similar: iteration in glob or the FS.
3. Assume we find the one network and run it; suddenly the simulation barf something about a file the user never heard of.

And so on...

Reply:
The list of possible candidates is probably nice, though!